index.html

```diff
@@ -2,23 +2,140 @@
 layout: default
 title: Home
 ---
-<div class="jumbotron jumbotron-fluid">
-  <div class="container">
+<div id="overlay" onclick="off()">
+  <div>
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/">Home</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/find-help">Find Help</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="https://airtable.com/shrDrTCZQmDlCN8sg">
+          Get Involved
+        </a>
+      </li>
+    </ul>
+  </div>
```

Comment:
Should this go into navbar.html and then include navbar.html in the default layout, so that it's available on every page?